script.module.resolveurl/lib/resolveurl/plugins/pixeldrain.py

```diff
@@ -54,8 +56,9 @@ def get_media_url_list(self, host, media_id):
         if(file_list['success'] is True):
             sources = []
             if(file_list['files']):
-                for file in file_list['files']:
-                    sources += [(file['name'], 'http://' + host + '/api/file/' + file['id'])]
+                for file_info in file_list['files']:
+                    if('video' in file_info['mime_type']):
+                        sources += [(file_info['name'], 'http://' + host + '/api/file/' + file_info['id'])]
```

Comment:
Can you please give me a sample embed. the lines from 56 -61 seem unduly complex and seems could be done in a single line. Would know for certain once I see a sample embed. Thanks